openshift/s2i/nginx-runtime/nginx.conf.template

```diff
@@ -39,6 +39,10 @@ http {
         underscores_in_headers on;
         ignore_invalid_headers off;
 
+        ssl_protocols               TLSv1.2 TLSv1.3;
```

Comment:
I'm curious if this'll work, we have more strict policies on our Openshift server right now but they appear to be getting overwritten by the SSL ciphers on the reverse proxy server (ie. vanity url).

Reply:
For reference:
![image](https://user-images.githubusercontent.com/41091511/220796016-ace21e5a-925f-4b4c-a5a3-5d7e5a93d822.png)

Reply:
and vanity url route: https://www.ssllabs.com/ssltest/analyze.html?d=dev-pims.th.gov.bc.ca

Reply:
ah, that explains some things. I will dig some more on the cipher's part. But at this point, I believe the spike cap has been reached.